docs/en/observability/whats-new.asciidoc

```diff
@@ -3,24 +3,94 @@
-// {observability-guide-all}/8.6/whats-new.html[8.6] |
-// {observability-guide-all}/8.5/whats-new.html[8.5] |
-// {observability-guide-all}/8.4/whats-new.html[8.4] |
-// {observability-guide-all}/8.3/whats-new.html[8.3] |
-// {observability-guide-all}/8.2/whats-new.html[8.2] |
-// {observability-guide-all}/8.1/whats-new.html[8.1] |
-// {observability-guide-all}/8.0/whats-new.html[8.0] |
-// {observability-guide-all}/7.17/whats-new.html[7.17]
+Other versions:
+{observability-guide-all}/8.12/whats-new.html[8.12] |
+{observability-guide-all}/8.11/whats-new.html[8.11] |
+{observability-guide-all}/8.10/whats-new.html[8.10] |
+{observability-guide-all}/8.9/whats-new.html[8.9] |
+{observability-guide-all}/8.8/whats-new.html[8.8] |
+{observability-guide-all}/8.7/whats-new.html[8.7] |
+{observability-guide-all}/8.6/whats-new.html[8.6] |
+{observability-guide-all}/8.5/whats-new.html[8.5] |
+{observability-guide-all}/8.4/whats-new.html[8.4] |
+{observability-guide-all}/8.3/whats-new.html[8.3] |
+{observability-guide-all}/8.2/whats-new.html[8.2] |
+{observability-guide-all}/8.1/whats-new.html[8.1] |
+{observability-guide-all}/8.0/whats-new.html[8.0] |
+{observability-guide-all}/7.17/whats-new.html[7.17]
 
 // tag::whats-new[]
 
-// What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)
+Elastic Observability 8.13 announces general availability (GA) for AWS Bedrock support in Elastic AI Assistant, along with enhancements for the AI Assistant and SLOs.
+
+[float]
+[[ai-assistant-enhancements]]
+== AI Assistant enhancements
+
+* Elastic AI Assistant support for AWS Bedrock
+* Contextual awareness of the entry point into AI Assistant
+* Performance improvements
+* Tech preview of editing and visualizing ES|QL queries from the AI Assistant
+
+[float]
+[[aws-bedrock-support]]
+=== AWS Bedrock support
+
+Elastic Observability 8.13 adds connectivity for AWS Bedrock, specifically for the Claude models. The Bedrock/Claude LLM adapter parses data from AWS using a SerDe stream using `@smithy/eventstream-serde-nodeuses` and offers the same experience as the other Elastic LLM connectors (OpenAI and Azure OpenAI). Setup is simple with the Amazon Bedrock connector configuration page.
+
+image::images/1-connectors.png[aws bedrock support connector config page]
+
+
+[float]
+[[entry-point-contextual-awareness]]
+=== Entry point contextual awareness
+
+When entering AI Assistant chat, it will now understand what was on the screen you entered it from. For 8.13, we've added screen context for the SLO list page, the SLO detail page, the alert details page, and the APM service inventory.
+
+image::images/2-nginx.png[slo entry point contextual awareness]
+
+[float]
+[[es-ql-editing-visualization]]
+=== ES|QL editing and visualization (tech preview)
+
+When the AI Assistant responses use ES|QL, you can now do the following:
+
+1. See the “Display results” of the ES|QL query
+2. Visualize the query on the generated ES|QL:
```

Comment:
Wasn't sure if this should be an unordered list or if it's actually a process of seeing the results then visualizing the query.

Reply:
```suggestion
* See the “Display results” of the ES|QL query
* Visualize the query on the generated ES|QL:
```